xchange-gdax/pom.xml

```diff
@@ -29,7 +29,14 @@
             <artifactId>xchange-core</artifactId>
             <version>${project.version}</version>
         </dependency>
-
+        
+        <dependency>
+        	<groupId>org.projectlombok</groupId>
+        	<artifactId>lombok</artifactId>
+        	<version>1.18.2</version>
+        	<scope>provided</scope>
+        </dependency>
```

Comment:
The dependency should be added to the parent pom under dependencyManagement

Reply:
@timmolter 

Tim -- I'm unsure how to do this correctly.  I've moved the dependency to the parent pom but I'm unable to build it either under eclipse or pushing it, maybe because the gdax pom points to the parent 4.3.10-SNAPSHOT, not to my new parent.

But if I change that, presumably I break something else?  What's the correct way to proceed here please?

Thanks.